src/Razor/src/Microsoft.VisualStudio.RazorExtension/source.extension.vsixmanifest

```diff
@@ -33,11 +33,11 @@
     <Asset Type="Microsoft.VisualStudio.Assembly" Path="Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X.dll" />
     <Asset Type="Microsoft.VisualStudio.Assembly" Path="Microsoft.AspNetCore.Mvc.Razor.Extensions.Version2_X.dll" />
     <Asset Type="Microsoft.VisualStudio.Assembly" Path="Microsoft.AspNetCore.Razor.Language.dll" />
+    <Asset Type="Microsoft.VisualStudio.Assembly" Path="MediatR.dll" />
     <Asset Type="Microsoft.VisualStudio.Assembly" Path="OmniSharp.Extensions.JsonRpc.dll" />
     <Asset Type="Microsoft.VisualStudio.Assembly" Path="OmniSharp.Extensions.LanguageProtocol.dll" />
     <Asset Type="Microsoft.VisualStudio.Assembly" Path="OmniSharp.Extensions.LanguageServer.dll" />
-    <Asset Type="Microsoft.VisualStudio.Assembly" Path="MediatR.dll" />
-    <Asset Type="Microsoft.VisualStudio.Assembly" Path="MediatR.Extensions.Microsoft.DependencyInjection.dll" />
```

Comment:
Why remove this?

Reply:
No longer used, DryIoc supports MediatR with a few lines of code.

Reply:
Oh and the DryIoc is using `DryIoc.Internal` so there are no new assemblies along with that.